### ISSUE TITLE
training loss is very low (0.177622) at the very beginning, is it normal?

### ISSUE THREAD

**Issue by sureone** [facebookresearch/SlowFast#159]
Thanks for the nice work, I found the loss is very low from the start of training, is it normal?

below are my training log:

`[INFO: logging.py:   67]: json_stats: {"_type": "train_iter", "cur_epoch": "1", "cur_iter": "10", "eta": "12:18:36", "loss": 0.177622, "lr": 0.100000, "mode": "train", "time_diff": 0.961633}
[INFO: logging.py:   67]: json_stats: {"_type": "train_iter", "cur_epoch": "1", "cur_iter": "20", "eta": "1 day, 14:35:48", "loss": 0.168755, "lr": 0.100000, "mode": "train", "time_diff": 3.015697}
[INFO: logging.py:   67]: json_stats: {"_type": "train_iter", "cur_epoch": "1", "cur_iter": "30", "eta": "12:21:22", "loss": 0.171540, "lr": 0.100000, "mode": "train", "time_diff": 0.965657}
[INFO: logging.py:   67]: json_stats: {"_type": "train_iter", "cur_epoch": "1", "cur_iter": "40", "eta": "2 days, 22:05:33", "loss": 0.103660, "lr": 0.100000, "mode": "train", "time_diff": 5.478963}
[INFO: logging.py:   67]: json_stats: {"_type": "train_iter", "cur_epoch": "1", "cur_iter": "50", "eta": "1 day, 8:26:06", "loss": 0.133752, "lr": 0.100000, "mode": "train", "time_diff": 2.535917}
[INFO: logging.py:   67]: json_stats: {"_type": "train_iter", "cur_epoch": "1", "cur_iter": "60", "eta": "1 day, 5:14:49", "loss": 0.113067, "lr": 0.100000, "mode": "train", "time_diff": 2.287157}
[INFO: logging.py:   67]: json_stats: {"_type": "train_iter", "cur_epoch": "1", "cur_iter": "70", "eta": "2 days, 6:35:12", "loss": 0.098588, "lr": 0.100000, "mode": "train", "time_diff": 4.269696}
[INFO: logging.py:   67]: json_stats: {"_type": "train_iter", "cur_epoch": "1", "cur_iter": "80", "eta": "3 days, 8:31:52", "loss": 0.111157, "lr": 0.100000, "mode": "train", "time_diff": 6.300399}
[INFO: logging.py:   67]: json_stats: {"_type": "train_iter", "cur_epoch": "1", "cur_iter": "90", "eta": "12:22:50", "loss": 0.102716, "lr": 0.100000, "mode": "train", "time_diff": 0.968811}
[INFO: logging.py:   67]: json_stats: {"_type": "train_iter", "cur_epoch": "1", "cur_iter": "100", "eta": "1 day, 15:23:25", "loss": 0.100519, "lr": 0.100000, "mode": "train", "time_diff": 3.083061}
`

**Comment by takatosp1** [comment]
Hi
Thanks for playing with PySlowFast, could you let me know what is the model you were training and what is the configs? (more specifically what is the batch size and how many gpu(s) did you use)

Nit, your time_diff looks a bit slow.

**Comment by sureone** [comment]
I use the config "configs/AVA/c2/SLOWFAST_32x2_R50.yaml", BATCH_SIZE is 4  with only 1 GPU to train(GTX1080)
